src/mlpack/methods/reinforcement_learning/q_networks/simple_dqn.hpp

```diff
@@ -108,7 +108,7 @@ class SimpleDQN
    *
    * @param state The input state.
    * @param target The training target.
-   * @return gradient The gradient.
+   * @param gradient The gradient.
    */
   void Backward(const arma::mat state, arma::mat& target,
 arma::mat& gradient)
```

Comment:
```suggestion
  void Backward(const arma::mat state, arma::mat& target, arma::mat& gradient)
```
Huh, wonder how this ended up on two lines? :)